game-core/src/main/java/games/strategy/engine/framework/ui/SaveGameFileChooser.java

```diff
@@ -50,7 +50,17 @@
 
     AUTOSAVE_ODD(getOddRoundAutoSaveFile(true)),
 
-    AUTOSAVE_EVEN(getEvenRoundAutoSaveFile(true));
+    AUTOSAVE_EVEN(getEvenRoundAutoSaveFile(true)),
+
+    AUTOSAVE_END_TURN(getBeforeStepAutoSaveFile("EndTurn", true)),
+
+    AUTOSAVE_BEFORE_BATTLE(getBeforeStepAutoSaveFile("Battle", true)),
+
+    AUTOSAVE_AFTER_BATTLE(getBeforeStepAutoSaveFile("Battle", true)),
```

Comment:
Should this be using `getAfterStepAutoSaveFile()`?

Reply:
Oops! Sorry about that boo boo.

Hopefully next time I'll get these things more correct.